hardware/2-0-revision.md

```diff
@@ -0,0 +1,272 @@
+| Speed         | 64MHz
+| Debug         | [SWD](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf52833%2Fdif.html), [J-Link/OB](https://www.segger.com/products/debug-probes/j-link/models/j-link-lite/j-link-lite-cortex-m/)
+| More Info     | [Software](../software), [NRF52 datasheet](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52833.html&cp=3_1)
+
+### Bluetooth Wireless Communication
+
+The on-board 2.4GHz supports Bluetooth communications via the [Nordic S113 SoftDevice](https://www.nordicsemi.com/Software-and-Tools/Software/S113), which provides a fully qualified Bluetooth low energy stack.
+This allows the micro:bit to communicate with a wide range of Bluetooth devices, including smartphones and tablets.
+
+| item          | details
+| ---           | ---
+| Stack         | Bluetooth 5.1 with Bluetooth Low Energy(BLE)
+| Band          | 2.4GHz ISM (Industrial, Scientific and Medical) 2.4GHz..2.41GHz
+| Channels      | 50 2MHz channels, only 40 used (0 to 39), 3 advertising channels (37,38,39)
+| Sensitivity   | -93dBm in Bluetooth low energy mode
+| Tx Power      | -40dBm to 4dBm
+| Role          | [GAP Peripheral & GAP Central](https://bluetooth-developer.blogspot.com/2016/07/microbit-and-bluetooth-roles.html)
+| Congestion avoidance | Adaptive Frequency Hopping
+| Profiles      | [BBC micro:bit profile](https://lancaster-university.github.io/microbit-docs/ble/profile/)
+| More Info     | [Bluetooth](../bluetooth)
+
+### Low level radio communications
+
+The on-board 2.4GHz transceiver supports a number of other radio communications standards, on which we build the microbit-radio protocol
+This protocol provides a very simple small-packet broadcast radio interface between other devices that support it, such as other micro:bit devices.
+The 'radio' interface that appears in a number of the languages on the micro:bit is built on top of this protocol.
+Additionally, the micro:bit runtime software adds a 'group code' to each data payload, allowing for simple user managed device addressing and filtering to take place.
+
+| item          | details
+| ---           | ---
+| Protocol      | [Micro:bit Radio](https://lancaster-university.github.io/microbit-docs/ubit/radio)
+| Freq band     | 2.4GHz
+| Channel rate  | 1Mbps or 2Mbps
+| Encryption    | None
+| Channels      | 80 (0..80)
+| Group codes   | 255
+| Tx power      | Eight user configurable settings from 0(-30dbm) to 7 (+4dbm)
+| Payload size  | 32 (standard) 255 (if reconfigured)
+| More Info     | [Micro:bit Radio](https://lancaster-university.github.io/microbit-docs/ubit/radio)
+
+### Buttons
+
+The two buttons on the front of the micro:bit, and the one button on the back, are tact momentary push-to-make buttons. The back button is connected to the KL27 interface processor and to the NRF52 processor for system reset purposes. This means that the application will reset regardless of if it is powered from USB or from battery.
```

Comment:
I'm adding a comment in this file for V2.0 because I cannot add a comment to this line in the V2.2 doc, but the V2.2 version needs to remove the KL27 reference.